woocommerce/woocommerce-ios
# Store onboarding > custom domains task: navigate to custom domains

Closes: #8909 


## Description


This PR handled the navigation when the user taps on the custom domains onboarding task. To show custom domains, `StoreOnboardingCoordinator` creates and starts `DomainSettingsCoordinator` with a new source with analytics event property `onboarding` (ref pe5sF9-15a-p2#comment-1694). `Site` is a necessary parameter, from looking around `ServiceLocator.stores.sessionManager.defaultSite` seems like how we get the latest site.

## Testing instructions


- Login if needed
- Go to the My store tab if needed
- Tap on `Customize your domain` row --> the domain settings modal should be shown
- Dismiss the modal
- Tap on any other row to see the fullscreen onboarding view
- Tap on `Customize your domain` row --> the domain settings modal should be shown

## Screenshots



https://user-images.githubusercontent.com/1945542/221117672-8b7aa0ba-f6a5-499a-a06d-4882dc8c9393.mov

portrait | landscape
-- | --
![Simulator Screen Shot - iPhone 14 - 2023-03-01 at 09 34 16](https://user-images.githubusercontent.com/1945542/222022527-758cf7c7-7547-4636-97ff-b893a33361c3.png) | ![Simulator Screen Shot - iPhone 14 - 2023-03-01 at 09 34 29](https://user-images.githubusercontent.com/1945542/222022531-7f23c95c-7574-47e4-8795-f3d32b000e06.png)
![Simulator Screen Shot - iPhone 14 - 2023-03-01 at 09 37 11](https://user-images.githubusercontent.com/1945542/222022537-6a6dbca8-9a9a-4455-9760-d3be65ecba7c.png) | ![Simulator Screen Shot - iPhone 14 - 2023-03-01 at 09 37 13](https://user-images.githubusercontent.com/1945542/222022540-a5d1341e-bb97-4183-a2ee-cff400298050.png)


---
